src/Gui/Camera.cpp

```diff
@@ -0,0 +1,200 @@
+/***************************************************************************
+ *   Copyright (c) 2023 Werner Mayer <wmayer[at]users.sourceforge.net>     *
+ *                                                                         *
+ *   This file is part of the FreeCAD CAx development system.              *
+ *                                                                         *
+ *   This library is free software; you can redistribute it and/or         *
+ *   modify it under the terms of the GNU Library General Public           *
+ *   License as published by the Free Software Foundation; either          *
+ *   version 2 of the License, or (at your option) any later version.      *
+ *                                                                         *
+ *   This library  is distributed in the hope that it will be useful,      *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU Library General Public License for more details.                  *
+ *                                                                         *
+ *   You should have received a copy of the GNU Library General Public     *
+ *   License along with this library; see the file COPYING.LIB. If not,    *
+ *   write to the Free Software Foundation, Inc., 59 Temple Place,         *
+ *   Suite 330, Boston, MA  02111-1307, USA                                *
+ *                                                                         *
+ ***************************************************************************/
```

Comment:
These files are still using the old-style copyright header, instead of the new one. Consider using the updated version in new files. Example here: https://github.com/FreeCAD/FreeCAD/blob/2d3953f65fff0db0bfa23d9877601a828ba24a7a/src/App/IndexedName.h#L1

Reply:
So, the copyright text is fine and only the SPDX identifier is missing?

Reply:
Sorry, GitHub doesn't make it easy to link to the whole code block. You've still got the version that references a street address, "59 Temple Place, Suite 330, Boston MA" instead of the URL they suggest now.

Reply:
Moved to new license header.